src/net_processing.cpp

```diff
@@ -867,7 +871,7 @@ static uint256 most_recent_block_hash;
 static bool fWitnessesPresentInMostRecentCompactBlock;
 
 /**
- * Maintain state about the best-seen block and fast-announce a compact block 
+ * Maintain state about the best-seen block and fast-announce a compact block
```

Comment:
Commit "[net] Tighten scope in net_processing"

Could not include these changes.